README.md

```diff
@@ -572,6 +572,7 @@ Usage of kube2iam:
       --metrics-port string                   Metrics server http port (default: same as kube2iam server port) (default "8181")
       --namespace-key string                  Namespace annotation key used to retrieve the IAM roles allowed (value in annotation should be json array) (default "iam.amazonaws.com/allowed-roles")
       --cache-resync-period                   Refresh interval for pod and namespace caches
+      --deal-with-duplicated-ip-in-cache       Queries the k8s api server when the pod's cache contains more than a pod with the same IP
```

Comment:
the purpose of this change is to deal with duplicated IPs in the local agent cache not to allow them. 
We don't want in any way have pod that share the same IP because this will create ambiguity on what role attach to it.
I think it is more expressive the name `deal-with-duplicated-ip-in-cache`

Reply:
"deal" is a [very vague word](https://www.merriam-webster.com/dictionary/deal). As a personal anecdote, I've never seen the word `deal` in a command line argument name for any script I've ever used. Perhaps we could find a more specific verb that conveys the intent.

At a higher level, perhaps this behavior should be enabled with a feature flag? Are there scenarios where users would explicitly not want this feature? It seems like something that could be a feature flag with a default of disabled, then after a few releases default it to enabled and finally remove the feature flag so it is permanently enabled.

Reply:
I'm open to suggestions for renaming the flag... i'm using `deal` because i'm dealing with duplicates in the pods cache :D and if there is a duplicate in the cache I'm reading from the api server to get the most update info.

maybe `--resolve-duplicated-ips-using-api` ?

The flag is off by default, if you run with AWS CNI plugin and you have short lived containers you want to enable this.

The only drawback of this flags are one off requests to the k8s api server, that cannot be tracked via the new Prometheus metrics this PR adds as well.